sway/criteria.c

```diff
@@ -46,6 +46,31 @@ static int regex_cmp(const char *item, const pcre *regex) {
 	return pcre_exec(regex, NULL, item, strlen(item), 0, 0, NULL, 0);
 }
 
+static int cmp_urgent(const void *_a, const void *_b) {
+	struct sway_view *a = *(void **)_a;
```

Comment:
Minor point, but don't cast away the const here.

Reply:
I copied this from `container_sort_workspaces`. When I tried it without the cast something weird was happening and it wasn't comparing them properly.